geniza/corpus/templates/corpus/snippets/document_tabs.html

```diff
@@ -2,7 +2,7 @@
 <!-- document detail page navigation -->
 {# Translators: accessibility text label for document detail view tabs navigation #}
 <nav aria-label="{% translate "tabs" %}">
-    <ul class="tabs">
+    <ul class="tabs" {% if LANGUAGE_CODE == "ar" or LANGUAGE_CODE == "he" %}dir="rtl"{% endif %}>
```

Comment:
do we need to put the `rtl` here? can it not be set at a higher level of context, or not yet because of the mixed content and we're only doing components?

Reply:
Right, since we're only doing certain components they'll need to either be set RTL one at a time, or the whole page set to RTL and then individual elements set LTR until RTL is implemented. We could just do that for the few problematic elements, but it might take some time to figure out which those are. (The search results page seems to be an especially bad example!)

Reply:
It might also be fine to just have some things look a little broken until implemented…!

Reply:
IMO it's ok if they don't display correctly in the interim since we're not making this version public yet. I'd rather implement it the way we intend to have it work at the end than have to go back and refactor everything. (i.e., set rtl once at the top level based on language, and implement styles and overrides for the components as you work your way through them.)

What do you mean about the "few problematic elements" ?

Reply:
By "few problematic elements" I just meant: some things already are usable and work fine (like most of the content page and document detail page elements), but some look very messy without special styling applied (like most of the elements on the search results page)—so I considered the latter problematic!